data/Publications.json

```diff
@@ -3730,5 +3706,2795 @@
         "bibcode": "2016AJ....152...78L",
         "doi": null,
         "description": null
+    },
+    {
+        "name": "Legg17",
+        "bibcode": "2017ApJ...842..118L",
+        "doi": "10.3847/1538-4357/aa6fb5",
+        "description": "The Y-type Brown Dwarfs: Estimates of Mass and Age from New Astrometry, Homogenized Photometry, and Near-infrared Spectroscopy"
+    },
```

Comment:
`Legg17` was added correctly to the Publications table, so we actually don't need to modify this table anymore!!  Huzzah!